pkg/cli/cmd/certs.go

```diff
@@ -0,0 +1,162 @@
+package cmd
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aserto-dev/topaz/pkg/cli/cc"
+	"github.com/aserto-dev/topaz/pkg/cli/certs"
+)
+
+type CertsCmd struct {
+	List     ListCertsCmd     `cmd:"" help:"list dev certs"`
+	Generate GenerateCertsCmd `cmd:"" help:"generate dev certs"`
+	Trust    TrustCertsCmd    `cmd:"" help:"trust dev certs"`
+}
+
+const (
+	DefaultCertsDir = "~/.config/topaz/certs"
```

Comment:
How does this resolve on Windows? 

We had to fix a similar bug in the default policy root path, (https://github.com/aserto-dev/topaz/commit/22ea3258fb6ed418df6de3433efc193b2ad9027f) 

`~` expands when your terminal is either powershell or  pwsh but not when using cmd.exe

Reply:
That is just a string defined as a constant for default cert path value (not used to resolve paths), in code we use the os.UserHomeDir() call to establish the home directory.